graphql-java-servlet/src/main/java/graphql/kickstart/servlet/HttpRequestHandlerImpl.java

```diff
@@ -55,4 +56,20 @@ public void handle(HttpServletRequest request, HttpServletResponse response) thr
       throw t;
     }
   }
+
+  private GraphQLInvocationInput parseInvocationInput(
+      HttpServletRequest request,
+      HttpServletResponse response) {
+    try {
+      GraphQLInvocationInputParser invocationInputParser =
+          GraphQLInvocationInputParser.create(
+              request,
+              configuration.getInvocationInputFactory(),
+              configuration.getObjectMapper(),
+              configuration.getContextSetting());
+      return invocationInputParser.getGraphQLInvocationInput(request, response);
+    } catch (Exception e) {
+      throw new InvocationInputParseException(e);
```

Comment:
Haven't checked the `getGraphQLInvocationInput` method itself, but it looks like it would make sense to move catching an exception and wrapping it in this custom `InvocationInputParseException` inside that `getGraphQLInvocationInput` method.

Reply:
Ah ok, clear. That's why I mentioned I hadn't looked at the code :)